specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/bigDataPool.json

```diff
@@ -437,6 +437,13 @@
           "$ref": "#/definitions/LibraryRequirements",
           "description": "Library version requirements"
         },
+        "customLibraries": {
```

Comment:
Normal advice is to change the API version when adding new properties to an existing API. It's not a hard requirement, but it can break down-level clients, depending on the service behavior. For example:
1. Portal or new client PUTs the resource, including the new properties
2. Downlevel client PUTs/PATCHs the same resource without the new properties.

What state will the resource be in afterward? Will it make sense to customers from the perspective of both clients and the portal?
Without incrementing the version, your service can't control that situation, e.g. rejecting down-level PUT/PATCH.

Reply:
This is a preview feature and will be rolled out in preview mode only. A new version will be created once this feature gets rolled out for customers.